bot/constants.py

```diff
@@ -113,4 +113,34 @@ def get_days(year: int, month: int):
     {"name": "minutes", "range": 59},
 ]
 
+SAFETY_API_ENDPOINT = "https://discordsafetyportal.zendesk.com/api/v2/help_center/articles/search"
+SAFETY_IDS = [
```

Comment:
Consider adding commentary for how/why this list is maintained the way it is. In other words, why is this list in this specific order, and where should we add new entries (if any)?

Reply:
Added comments to explain each ID with their current name, and why it is in this specific order.